src/main/java/org/opensrp/service/EventService.java

```diff
@@ -24,7 +23,9 @@
 import org.springframework.beans.factory.annotation.Value;
 import org.springframework.stereotype.Service;
 
-import com.fasterxml.jackson.core.JsonProcessingException;
+import java.text.SimpleDateFormat;
+import java.util.*;
```

Comment:
Let's avoid wildcard imports

Reply:
I think then we may need to decide about updating this setting in the OpenMRS fromatter. Codacy was also against applying wildcards to imports, whereas the formatter applies wildcard when the imports count is greater than 5.

Reply:
I agree let's update the formatter to stop adding the wildcards

Reply:
I replaced the wildcard with qualified imports in the meantime